pkg/mcs/scheduling/server/cluster.go

```diff
@@ -309,11 +309,12 @@ func (c *Cluster) updateScheduler() {
 		)
 		// Create the newly added schedulers.
 		for _, scheduler := range latestSchedulersConfig {
+			schedulerType := types.ConvertOldStr2Type[scheduler.Type]
```

Comment:
How does ConvertOldStrToType sound to you? Variable names with embedded numbers are not common in PD.

Reply:
They are same for me. I can update it